### PR TITLE
remove some specific php5.5 code

### PR DESCRIPTION
**Pull request by antograssiot**

### CODE REVIEW

File: tests/TestCase/Database/Driver/SqliteTest.php [hunk 1]
```diff
@@ -127,11 +127,7 @@ public static function schemaValueProvider()
     public function testSchemaValue($input, $expected)
     {
         $driver = new Sqlite();
-        $pdo = PDO::class;
-        if (version_compare(PHP_VERSION, '5.6', '<')) {
-            $pdo = 'FakePdo';
```
Comment (inoas) [inline tests/TestCase/Database/Driver/SqliteTest.php]: What is `FakePdo` and if it is class source code, is still required now?

**Comment by ADmad** [comment]
@antograssiot PHP 7.1 builds are segfaulting :(  Did they work before this PR?

File: tests/TestCase/Database/Driver/SqliteTest.php (hunk 1 above)
Comment (antograssiot) [inline tests/TestCase/Database/Driver/SqliteTest.php]: it doesn't exists @ionas nothing to remove here

**Comment by antograssiot** [comment]
@ADmad, they are segfaulting since last night, travis updated to from beta2 to php7.1-beta3 :(
I commented in another PR that it was maybe too early/optimist of me to remove those build from the allowed to fail matrix.
I can update travis until it becomes stable

**Comment by ADmad** [comment]
@antograssiot Okay, let's put back 7.1 under allow_failures.

**Pull request merged**